cosmos/providers/databricks/plugin.py

```diff
@@ -0,0 +1,279 @@
+    `RunsApi` class does not allow sending the `include_history` parameter which is necessary for
+    repair jobs.
+
+    Also for the moment we don't allow custom retry_callbacks. We might implement this in
+    the future if users ask for it, but for the moment we want to keep things simple while the API
+    stabilizes.
+
+    :param databricks_conn_id: The Databricks connection ID.
+    :param databricks_run_id: The Databricks run ID.
+    :param tasks_to_repair: A list of Databricks task IDs to repair.
+    :return: None
+    """
+
+    def _get_api_client():
+        hook = DatabricksHook(databricks_conn_id)
+        databricks_conn = hook.get_conn()
+        return ApiClient(
+            user=databricks_conn.login,
+            token=databricks_conn.password,
+            host=databricks_conn.host,
+        )
+
+    api_client = _get_api_client()
+    jobs_service = JobsService(api_client)
+    current_job = jobs_service.get_run(run_id=databricks_run_id, include_history=True)
+    repair_history = current_job.get("repair_history")
+    repair_history_id = None
+    if repair_history and len(repair_history) > 1:
+        # We use the last item in the array to get the latest repair ID
+        repair_history_id = repair_history[-1]["id"]
+    return jobs_service.repair(
+        run_id=databricks_run_id,
+        version="2.1",
+        latest_repair_id=repair_history_id,
+        rerun_tasks=tasks_to_repair,
+    )
+
+
+class DatabricksJobRunLink(BaseOperatorLink):
+    """Constructs a link to monitor a Databricks Job Run."""
+
+    name = "See Databricks Job Run"
+
+    def get_link(
+        self,
+        operator: BaseOperator,
+        dttm=None,
+        *,
+        ti_key: TaskInstanceKey | None = None,
+    ) -> str:
+        dag = get_airflow_app().dag_bag.get_dag(ti_key.dag_id)
+        dag.get_task(ti_key.task_id)
+        # Should we catch the exception here if there is no return value?
+        metadata = XCom.get_one(
+            task_id=ti_key.task_id,
+            dag_id=ti_key.dag_id,
+            run_id=ti_key.run_id,
+            key="return_value",
+        )
+        hook = DatabricksHook(metadata.databricks_conn_id)
+        return f"https://{hook.host}/#job/{metadata.databricks_job_id}/run/{metadata.databricks_run_id}"
+        return f"/foo/bar?dag_id={operator.dag_id}&task_id={operator.task_id}"
+
+
+class DatabricksJobRepairAllFailedLink(BaseOperatorLink):
+    """Constructs a link to send a request to repair all failed databricks tasks."""
+
+    name = "Repair All Failed Tasks"
+
+    def get_link(
+        self,
+        operator,
+        dttm=None,
+        *,
+        ti_key: TaskInstanceKey | None = None,
+    ) -> str:
+        # Should we catch the exception here if there is no return value?
+        metadata = XCom.get_one(
+            task_id=ti_key.task_id,
+            dag_id=ti_key.dag_id,
+            run_id=ti_key.run_id,
+            key="return_value",
+        )
```

Comment:
```suggestion
        metadata = XCom.get_one(
            ti_key=ti_key,
            key="return_value",
        )
```
same suggestion as above